src/components/UserIntroductionEditor/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { useForm } from 'react-hook-form';
+
+import { DONE_BUTTON_TEXT } from '@/constants/profile';
+import { PropsIntroductionEditor } from '@/types/profile';
+
+import HookFormInput from '../Common/HookFormInput';
+import {
+  Introduction,
+  IntroductionBar,
+  IntroductionButton,
+  IntroductionForm,
+  IntroductionWrapper,
+} from './style';
+
+export default function UserIntroductionEditor({
+  isEditing,
+  onFormSubmit,
+  placeholderText,
+  onInputChange,
+  introduction,
+  onEditButtonClick,
+  buttonText,
+}: PropsIntroductionEditor) {
+  const { register } = useForm();
+
+  return (
+    <>
+      {isEditing ? (
+        <>
+          <IntroductionForm onSubmit={onFormSubmit}>
+            <HookFormInput
+              name="content"
+              placeholder={placeholderText}
+              value={introduction}
+              type="text"
+              style={{
+                height: '3.2rem',
+                width: '26.2rem',
+                padding: '0.4rem 0.8rem',
```

Comment:
```suggestion
                border: 'none',
                borderBottom: '1px solid #f3f2f2',
                paddingBottom: '1rem',
```
`input`을 이렇게 수정해도 좋을거 같아요
수현님 `input` 스타일링하신대로 통일하는게 좋을거 같습니다!